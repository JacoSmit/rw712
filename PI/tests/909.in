10
7
64 98 40 97 74 78 88
7
56 59 76 81 58 75 86
7
1 19 56 92 36 12 3
7
17 84 29 88 81 31 31
7
4 71 55 25 30 72 26
7
52 75 95 79 59 39 100
7
65 20 94 99 95 85 31
7
34 18 59 16 32 76 45
7
51 25 14 2 18 43 29
7
44 87 59 97 21 34 76
