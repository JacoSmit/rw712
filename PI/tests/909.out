1:
1 ~ 16
2 ~ 24
3 ~ 16
4 ~ 24
5 ~ 16
6 ~ 20
7 ~ 20
2:
1 ~ 20
2 ~ 20
3 ~ 20
4 ~ 20
5 ~ 20
6 ~ 20
7 ~ 20
3:
1 ~ 0
2 ~ 16
3 ~ 16
4 ~ 48
5 ~ 16
6 ~ 0
7 ~ 0
4:
1 ~ 6
2 ~ 30
3 ~ 6
4 ~ 30
5 ~ 30
6 ~ 6
7 ~ 6
5:
1 ~ 0
2 ~ 32
3 ~ 24
4 ~ 8
5 ~ 8
6 ~ 32
7 ~ 8
6:
1 ~ 12
2 ~ 20
3 ~ 24
4 ~ 20
5 ~ 16
6 ~ 12
7 ~ 28
7:
1 ~ 22
2 ~ 2
3 ~ 22
4 ~ 26
5 ~ 26
6 ~ 22
7 ~ 2
8:
1 ~ 16
2 ~ 8
3 ~ 24
4 ~ 8
5 ~ 16
6 ~ 36
7 ~ 16
9:
1 ~ 38
2 ~ 14
3 ~ 6
4 ~ 2
5 ~ 10
6 ~ 26
7 ~ 18
10:
1 ~ 14
2 ~ 26
3 ~ 18
4 ~ 30
5 ~ 6
6 ~ 10
7 ~ 22
